### DISCUSSION TITLE
Getting it to work on OSX

### DISCUSSION THREAD

**Discussion by skoop82** [karloskalcium/md2enex#9] [Q&A] [answered]
Sorry, I'm a complete coding and command line ignoramus, but I could really use the tool you have created for converting md files to enex (the lack of export options in Apple Notes is outrageous). I just don't know how to install and get this tool to work on MacOS. I can't just type these things in to the terminal, right? If you can point me to any direction to get me started, I would be very thankful.

**Comment by karloskalcium** [comment, marked as answer]
Instructions are [here](https://github.com/karloskalcium/md2enex?tab=readme-ov-file#install-and-run).

You'll need to install a supported version of python, depending on which version of OSX you have. Then you install from the command line by just cloning the repo and then running the commands in terminal.

I'd suggest considering using [Exporter](http://falcon.star-lord.me/exporter/) to export notes from Apple notes.

Here's a general terminal intro I found that may be useful: https://www.freecodecamp.org/news/command-line-for-beginners/
